_posts/2016-09-02-homebrew-1.0.0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 1.0.0
+author: MikeMcQuaid
+---
+Today I'm proud to announce the release of Homebrew 1.0.0. We've been working hard over the last year to make some major changes to Homebrew that we've been wanting for a long time. There's been some hiccups along the way but we now have a more stable base for using and developing Homebrew in future.
+
+These include:
+
+- [Homebrew package management and formulae (package metadata) split into separate repositories](https://github.com/Homebrew/brew/pull/2)
+- [Homebrew Community site (using Discourse) for discussion](http://community.brew.sh)
+- [Homebrew CI and homebrew/core use the OS X sandbox for build-time security](https://github.com/Homebrew/legacy-homebrew/pull/38361)
+- [Homebrew Cask integrated into Homebrew/brew](https://github.com/Homebrew/brew/pull/725)
+- [Add new `brew boneyard-formula-pr` command to create new formula version pull requests](https://github.com/Homebrew/brew/pull/53)
```

Comment:
I think you mislabelled this one? You've linked/referenced the boneyard command but seem to be talking about `bump-formula-pr` ❓